pkg/terra/report_fuzz_test.go

```diff
@@ -0,0 +1,35 @@
+//go:build go1.18
+// +build go1.18
+
+package terra
+
+import (
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/libocr/offchainreporting2/reportingplugin/median"
+)
+
+// go test -tags=go1.18 -fuzz ./...
+func FuzzMedianFromReport(f *testing.F) {
```

Comment:
Should the func name have `ReportCode` too?
```suggestion
func FuzzReportCodecMedianFromReport(f *testing.F) {
```